exercises/grep/example.py

```diff
@@ -42,9 +42,18 @@ def format_lines(matched_lines, files, flags):
     return ''.join(result)
 
 
-def grep(pattern, files, flags=''):
+def grep(pattern, files, flags=None):
+    """
+    Linux-like grep command.
+    :param pattern: [STRING]. Match pattern.
+    :param files: [LIST].
+    :param flags: [STRING]. Additional attributes.
+    :return: [STRING].
+    """
+    # Default value
+    if flags is None:
+        flags = ''
```

Comment:
What is the benefit of handling `flags` this way instead of just having a default value assigned in function arguments?